sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -114,15 +114,15 @@ class DateTime final : public _detail::Clock::time_point {
   }
 
   /**
-   * @brief Construct an instance of #Azure::DateTime from base class.
+   * @brief Constructs an instance of `DateTime` from a `time_point`.
```

Comment:
Why no `%` here? Can you please help me understand the difference.

Reply:
Without `%`, it inserts a link. They render just as you'd expect - without the `%` and not as hyperlink. You can look in the build artifacts - https://dev.azure.com/azure-sdk/public/_build/results?buildId=922992&view=artifacts&pathAsName=false&type=publishedArtifacts

Reply:
It should be there, I missed it!

Reply:
Will do a follow up PR.

Reply:
#2390

Reply:
I think a few other instances were missed. For example:
https://github.com/antkmsft/azure-sdk-for-cpp/blob/6185cba9e6701a1943f48f52963c963637cd8e52/sdk/core/azure-core/inc/azure/core/datetime.hpp#L52

https://github.com/antkmsft/azure-sdk-for-cpp/blob/6185cba9e6701a1943f48f52963c963637cd8e52/sdk/core/azure-core/inc/azure/core/datetime.hpp#L134-L136

https://github.com/antkmsft/azure-sdk-for-cpp/blob/6185cba9e6701a1943f48f52963c963637cd8e52/sdk/core/azure-core/inc/azure/core/datetime.hpp#L172

Reply:
Not fixing all of them so far, only the places I was adding. There are methods next to constructors, and they do still do self-reference. There are work items to go and update classes in each namespace, divided by the team member, it can be covered there. I'll go and update comments there.

Reply:
I updated #2299, #2300, and #2304